findbugs-exclude.xml

```diff
@@ -0,0 +1,5 @@
+<FindBugsFilter>
+     <Match>
+       <Class name="~.+\.Messages" />
```

Comment:
> I was getting Findbugs errors because of the generated `Messages` class

It is the responsibility of `maven-localizer-plugin` to generate sources which are free of FindBugs warnings. And last I checked, it does so. What did you find?